packages/handlersjs-http/lib/handlers/json-store-handler.spec.ts

```diff
@@ -0,0 +1,135 @@
+import { MemoryStore } from '@digita-ai/handlersjs-core';
+import { HttpHandlerContext } from '../models/http-handler-context';
+import { HttpHandlerResponse } from '../models/http-handler-response';
+import { HttpMethods } from '../models/http-method';
+import { JsonStoreHandler } from './json-store-handler';
+
+describe('JsonStoreHandler', () => {
+
+  interface StoreInterface {
+    data: string[];
+    anotherstring: number;
+  }
+
+  const inputData = [ 'abc', 'defghij', '123' ];
+  const inputData2 = [ ... inputData, '456' ]; // another list of data that may be used to update the store
+
+  let jsonStoreHandler: JsonStoreHandler<StoreInterface>;
+  let memoryStore: MemoryStore<StoreInterface>;
+  let requestContext: HttpHandlerContext;
+
+  beforeEach(() => {
+
+    memoryStore = new MemoryStore([ [ 'data', inputData ] ]);
+
+    jsonStoreHandler = new JsonStoreHandler('data', memoryStore);
+
+    requestContext = {
+      request: {
+        url: new URL('http://localhost/'),
+        method: 'GET',
+        headers: {},
+      },
+    };
+
+  });
+
+  describe('handle()', () => {
+
+    it('can GET the data correctly', async () => {
+
+      const response: HttpHandlerResponse = await jsonStoreHandler.handle(requestContext).toPromise();
+      const resultData: string[] = JSON.parse(response.body);
```

Comment:
Just curious as i cant find an answer to my question on google quickly:
I always use `await response.text()` instead of `response.body`, is there a difference that you know of?